Autopilot/Test/Inc/IMU_Mock.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef IMU_MOCK_HPP
+#define IMU_MOCK_HPP
+
+#include "gmock/gmock.h" // Brings in gMock
+
+#include "IMU.hpp"
+
+
+class MockIMU : public IMU {
+    public: 
+        MOCK_METHOD(void, Init, ());
+        MOCK_METHOD(void, Begin_Measuring, ());
+        MOCK_METHOD(void, GetResult, (IMUData_t *Data));
+
+};
+
+#endif
```

Comment:
Everything prior to c++ 11(which happens to be the standard we compile with, and why this doesn't cause warnings), requires a new line at the end of every file. May as well add one, for whatever backwards compatibility and so we don't see this ugly red circle during reviews. (You can get your text editor to add one automatically.)